src/components/DayPickerNavigation.jsx

```diff
@@ -33,6 +35,8 @@ const propTypes = forbidExtraProps({
 });
 
 const defaultProps = {
+  hasPrev: true,
```

Comment:
boolean props should default to false, so that absence and falsiness are the same. Can the naming of these be inverted?

Reply:
Agreed! Let's call this `disablePrev` and `disableNext` instead

Reply:
Fair point. I've come up with `isPrevMonthBlocked`/`isNextMonthBlocked`, to try and keep it consistent with `isDayBlocked` prop.